src/routes/+page.svelte

```diff
@@ -25,34 +25,92 @@
     </div>
 </section>
 
-<section class="px-6 lg:px-32 pt-24">
+<section class="px-6 lg:px-32 pt-12 pb-8">
     <div class="border-b border-gray-500 py-12 mb-8 lg:flex lg:justify-between">
-        <h2 class="flex items-center text-6xl font-interSemi tracking-tight pb-4">
+        <h2
+            class="flex items-center text-6xl font-interSemi tracking-tight pb-4"
```

Comment:
don't use `pb-4` in every element. apply `flex flex-col gap-y-4` to the parent element, and if any children needs a bit more space adjust these children accordingly